### PR TITLE
Addition of new Features

### PR DESCRIPTION
**Pull request by srvkmr130**
This completes the module 2 on writing Documentation for Anna.

### CODE REVIEW

File: docs/AdditionOfNewFeatures.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+#### Followings steps can be taken into consideration for the ADDITION OF NEW FEATURE  
+* Think something creativity that can be added as a feature.
+* Create the corrosponding INTENT in the API.
+* Tweak the code and assign the desired chrome operation to it.
+* Allot the permission in _manifest.json_ file,if the chrome operation needs any permission.
+* You are done.:+1:
+                                         
+**WE WOULD:heart:TO HEAR SOME INNOVATIVE FEATURES FROM YOU.**
+
```
Comment (gauthamzz) [inline docs/AdditionOfNewFeatures.md:51]: can you add like look in issues section for it and give a link

File: docs/AdditionOfNewFeatures.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Anna
+---
+### ADDITION OF NEW FEATURES
+
+To every new feature added in the Anna-Chrome Extension, first it must be declared as an **INTENT** in the corrosponding API agent.The background.js contains the main code and acts as a platform to addition of new features.
```
Comment (gauthamzz) [inline docs/AdditionOfNewFeatures.md:5]: It would be good to add a part which says how it works

File: docs/AdditionOfNewFeatures.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#### Followings steps can be taken into consideration for the ADDITION OF NEW FEATURE  
+* Think something creativity that can be added as a feature.
+* Create the corrosponding INTENT in the API.
+* Tweak the code and assign the desired chrome operation to it.
+* Allot the permission in _manifest.json_ file,if the chrome operation needs any permission.
+* You are done.:+1:
+                                         
+**WE WOULD:heart:TO HEAR SOME INNOVATIVE FEATURES FROM YOU.**
+
+
```
Comment (gauthamzz) [inline docs/AdditionOfNewFeatures.md:52]: It would also be nice to add some links for additional reading

**Comment by srvkmr130** [comment]
Done with the documentation part Module 2 for Anna.

**Review by gauthamzz — APPROVED** [review]

**Pull request merged**